android/MobiledgeXSDKDemo/build.gradle

```diff
@@ -1,13 +1,13 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
-project.ext.matchingengineVersion = "3.0-rc2-rebuild"
+project.ext.matchingengineVersion = "3.0-rc3"
```

Comment:
I don't if you need to update this to 3.0.0 or not, I think rc3 is 3.0.0

Reply:
It's actually 3.0 now, but basically exactly rc3.

Reply:
Updated to "3.0".